### PR TITLE
Fixes the console drawer pref so that it is persisted across sessions

### PR DESCRIPTION
**Pull request by jasonLaster**
small fix so that the drawer stays closed on refresh

### CODE REVIEW

**Review by markerikson — APPROVED** [review]

**Review by bvaughn — APPROVED** [review]

**Comment by bvaughn** [comment]
@jasonLaster Looks like [Playwright was still failing](https://github.com/replayio/devtools/actions/runs/3083780888/jobs/4985136186) for this PR. Did you mean to merge it as-is?

**Comment by bvaughn** [comment]
Reverting in #7771. This broke main.

File: packages/bvaughn-architecture-demo/components/console/ConsoleRoot.tsx [hunk 1]
```diff
@@ -22,6 +15,7 @@ import MessagesList from "./MessagesList";
 import Search from "./Search";
 import { SearchContextRoot } from "./SearchContext";
 import classNames from "classnames";
+import { useBoolPref } from "ui/hooks/settings";
```
Comment (bvaughn) [inline packages/bvaughn-architecture-demo/components/console/ConsoleRoot.tsx:18]: The Playwright tests were failing because this import is invalid.

**Comment by bvaughn** [comment]
Relanded in #7773

**Pull request merged**